HeterogeneousCore/CUDAUtilities/interface/HistoContainer.h

```diff
@@ -72,7 +72,7 @@ namespace cudautils {
   inline void launchFinalize(Histo *__restrict__ h,
                              uint8_t *__restrict__ ws
 #ifndef __CUDACC__
-                             = cudaStreamDefault
+                             = nullptr
```

Comment:
I am curious if using `cudaStreamDefault` was giving problems ?

Reply:
no,to me nullptr makes more sense for a pointer (even if they are all == 0 )